src/AWSCore.jl

```diff
@@ -203,7 +203,10 @@ Service endpoint URL for `request`.
 function service_url(aws, request)
     endpoint = get(request, :endpoint, request[:service])
     region = "." * aws[:region]
-    if endpoint == "iam" || (endpoint == "sdb" && region == ".us-east-1") || region=="."
+
+    regionless_service_endpoints = ["route53", "sdb"]
+
+    if endpoint == "iam" || (endpoint in regionless_service_endpoints && region == ".us-east-1") || region=="."
```

Comment:
I'm not loving the name `regionless_service_endpoints` as "iam" is truly regionless

Reply:
`endpoints_without_regions`?

Reply:
I think `regionless_endpoints` is a good name :+1: